gdal/ogr/ogrsf_frmts/amigocloud/ogramigocloudlayer.cpp

```diff
@@ -123,19 +123,7 @@ OGRFeature *OGRAmigoCloudLayer::BuildFeature(json_object* poRowObj)
             }
             else if( json_object_get_type(poVal) == json_type_string )
             {
-                if( poFeatureDefn->GetFieldDefn(i)->GetType() == OFTDateTime )
-                {
-                    OGRField sField;
-                    if( OGRParseXMLDateTime( json_object_get_string(poVal),
-                                             &sField) )
-                    {
-                        poFeature->SetField(i, &sField);
-                    }
-                }
-                else
-                {
-                    poFeature->SetField(i, json_object_get_string(poVal));
-                }
+                poFeature->SetField(i, json_object_get_string(poVal));
```

Comment:
this change is possible since SetField(int, const char*) properly handles ISO 8601 format, right ?

Reply:
Yes, it seems to be working properly now.